app.js

```diff
@@ -3,18 +3,25 @@ const express = require('express');
 require('express-async-errors');
 const logger = require('morgan');
 const cors = require('cors');
+const TokenValidator = require('twilio-flex-token-validator').validator;
 
 const swagger = require('./swagger');
 const { apiV0 } = require('./routes');
 
 const app = express();
 const apiKey = process.env.API_KEY;
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
```

Comment:
It would be worth [creating a new Twilio API key](https://www.twilio.com/console/project/api-keys) and testing this with that.  Then, if that key was ever compromised, we can just delete it.  It would also allow us the ability to rotate keys.  If testing that doesn't work, we can use the auth token.

Reply:
I've been reading docs and trying to achieve this, but I'm not sure if that's possible. This is the type of the `TokenValidator` (it requires the auth token).
![Screenshot from 2021-01-18 11-16-14](https://user-images.githubusercontent.com/15805319/104926465-f5d41380-597e-11eb-89a9-e6f2c16d0bd6.png)

It seems like having a [sub-account](https://support.twilio.com/hc/en-us/articles/360011348693-View-and-Create-New-Twilio-Subaccounts) can achieve the desired behavior, but a sub-account requires configuring all of the Tilio environment (task router, messaging, etc).

On the other hand, in Twilio console we can ask for a "secondary auth token". Maybe can try this?
![Screenshot from 2021-01-18 11-13-54](https://user-images.githubusercontent.com/15805319/104926509-04bac600-597f-11eb-918e-9e8458b0ceac.png)